internal/datastore/blockstore.go

```diff
@@ -45,6 +44,22 @@ import (
 // respective substores don't need to optimize or worry about Batching/Txn.
 // Hence the simplified ReaderWriter.
 
+func newCIDKeyBytes(c cid.Cid) ([]byte, error) {
+	sep := []byte("/")
```

Comment:
question: Why bother?  Why not drop the `"/"` completely?

Reply:
It's our prefix separator. We use that with every key.

Reply:
We dont need to, especially not for the blockstore, where every key is a single value